.env.example

```diff
@@ -1,13 +1,11 @@
-# Base Path Openshift: /business Firebase: /
 VUE_APP_PATH="/"
 
-#vaults Shared
+# vaults canadapost
 VUE_APP_ADDRESS_COMPLETE_KEY=
 
-#vaults web-url
+# vaults web-url
 VUE_APP_REGISTRY_HOME_URL="https://dev.bcregistry.gov.bc.ca/"
 VUE_APP_AUTH_WEB_URL="https://dev.account.bcregistry.gov.bc.ca/"
-VUE_APP_BUSINESSES_URL="https://dev.account.bcregistry.gov.bc.ca/"
```

Comment:
Obsolete.

Reply:
In fact, you'll notice that it's same URL as Auth Web URL above. This is because the BRD was previously part of Auth Web.

The old config worked because Auth Web redirected users to the BRD. Below, you'll see that we're now navigating users directly to the BRD.